.codeclimate.yml

```diff
@@ -0,0 +1,2 @@
+exclude_patterns:
+- "**/__test__/"
```

Comment:
@sebbss , this is good work but please add a new line to this file

Reply:
this has been resolved